src/business_logic/fact_state/in_memory_state_reader.rs

```diff
@@ -49,7 +49,7 @@ impl InMemoryStateReader {
         }
     }
 
-    fn get_compiled_class(
+    pub fn get_compiled_class(
```

Comment:
What is this change for?

Reply:
is useless, it was a try to check if i could use that function before being told that is not required/supported anymore